model/test/data-model.test.js

```diff
@@ -1,7 +1,7 @@
 import * as Code from '@hapi/code'
 import * as Lab from '@hapi/lab'
 
-import { Data } from '../client/model/data-model'
+import { Data } from '../src/data-model'
```

Comment:
should probably be using exported { Data } from (root) index.js?

Reply:
Fixed